### DISCUSSION TITLE
CR-MNZ1

### DISCUSSION THREAD

**Discussion by miomia** [Koenkk/zigbee2mqtt#23673] [Devices]
Hi  
The device CR-MNZ1 on the site it is stated that you are supported in Zigbee2mqqt but after adding to Home Assistants not everything is functional Monitoring does not work: current, voltage, power. Here is a link to a device-enabled site (https://www.zigbee2mqtt.io/devices/CR-MNZ1.html)
What to do?
Thanks 
<img width="891" alt="Снимок экрана 2024-08-20 в 17 11 23" src="https://github.com/user-attachments/assets/fc153341-c6c8-49d6-a953-bba11acb9e34">
<img width="415" alt="Снимок экрана 2024-08-20 в 17 12 48" src="https://github.com/user-attachments/assets/f2597913-78d7-4cde-bfae-73f80996bc3d">

**Comment by mstraa** [comment]
Hello, I'm new to HA in general but I found this : https://community.home-assistant.io/t/cr-mnz1-switch-with-power-monitoring/766807 which led to this https://github.com/klibro/zha-quirks/blob/839c3c6be09c3db5c0420974e6bad68951bead6b/TS0001_TZ3000_tgddlx4.py

I will take a look this week to see if I can do something, but any help will be appreciated :)

**Comment by rsteininger** [comment]
hi there @miomia and @mstraa,
Was the issue with "NULL" or "N/A" ever resolved? I have an older switch that works perfectly fine and now I had ordered 2 newer ones, which look identical to the old one BUT have the same issue as described above. 
Thanks!

**Reply by mstraa** [reply]
Sorry, my unit died few days after my post, so there will be nothing new on my part ^^
